agent/src/agent/pipeline/config/jython_scripts/create_events.py

```diff
@@ -6,13 +6,27 @@
 finally:
     sdc.importUnlock()
 
+
+def validate_event(event, record):
+    keys = [key for key in event]
+    if len(keys) == 0:
+        return False
+    for required_key in sdc.userParams.get("REQUIRED_FIELDS", []):
```

Comment:
and what happens when no data? the record will not be validated because when no data there is only last timespamp

Reply:
This method validates only events, if event have no filelds, than record contain only last timestamp, and we just skips this record